### PR TITLE
feat: Add package manager name

### PR DESCRIPTION
**Pull request by lwywoo**
Screenshot 2019-05-10 at 14 55 49

JIRA: https://snyksec.atlassian.net/browse/BST-79

Adding the first component of the ticket which is including the package manager in the vuln card.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,11 +20,14 @@
...
-    "marked": "^0.6.1",
+    "coveralls": "^3.0.3",
+    "handlebars": "^4.1.2",
+    "marked": "^0.6.2",
     "minimist": "^1.2.0",
     "moment": "^2.24.0",
-    "source-map-support": "^0.5.10"
+    "nyc": "^13.3.0",
+    "source-map-support": "^0.5.10",
+    "tap": "^12.1.0"
```
Comment (adrukh) [inline package.json]: Tiny comment here - let's put dev deps into `devDependencies`, such as `coveralls`, `nyc` and `tap`. Do we really need them all?

Comment (lwywoo) [reply package.json]: Moved coveralls, nyc and tap to devDeps

Comment (adrukh) [reply package.json]: Thanks!
I still wonder - we didn't use to have `nyc` and `coveralls` prior to this PR, and I don't see them used by any `script`. Do we need them then?

Comment (lwywoo) [reply package.json]: Seems like there might be some issue with Wizard, Kosta is looking into it.

Comment (lwywoo) [reply package.json]: Downgraded the cli to version 1.163.1 and then ran wizard, seems to have done the trick.

File: src/lib/snyk-to-html.ts [hunk 2]
```diff
@@ -93,7 +94,7 @@ function mergeData(dataArray: any[]): any {
   const totalDepCount =
     dataArray.reduce((acc, item) => acc + item.dependencyCount || 0, 0);
 
-  const paths = dataArray.map(d => d.path);
+  const paths = dataArray.map(d => ({path: d.path, packageManager: d.packageManager}));
```
Comment (miiila) [inline src/lib/snyk-to-html.ts]: I know it was already here, but I'd vote for renaming `d` to something meaningfull - I don't even know, what it is :-)

Comment (lwywoo) [reply src/lib/snyk-to-html.ts]: Thanks for reviewing, updated with a more sensible name.

**Pull request merged**